app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -56,10 +62,38 @@ export default function NoteTable({
               </td>
               <td>
                 <div className="flex flex-row items-center justify-start text-gray-900">
-                  {note.title}
+                  <a>{note.title}</a>
+                </div>
+              </td>
+              <td className="max-w-0 truncate">{note.description}</td>
+              <td className="text-center">
+                <Badge color={note.tag.color} size="small">
+                  {note.tag.tagTitle}
+                </Badge>
+              </td>
+              <td className="text-center">
+                {moment(note.created_at).format("MMM DD,YYYY")}
+              </td>
+              <td className="text-center">
+                {note.dueDate
+                  ? moment(note.dueDate).format("MMM DD,YYYY")
+                  : "--"}
+              </td>
+              <td className="text-center">
+                <div className="flex flex-row items-center justify-center">
+                  <Avatar size={36} contact={{ name: note.contact }} />
+                </div>
+              </td>
+              <td>
+                <div className="group-hover:opacity-100 opacity-0 flex flex-row space-x-4 items-center">
```

Comment:
Please remove hover effects since we are adding `nui-table--actions` for the same.